numba/decorators.py

```diff
@@ -77,6 +77,12 @@ def jit(signature_or_function=None, locals={}, target='cpu', cache=False, **opti
                 mode. Any arrays that the tight loop uses should be created
                 before the loop is entered. Default value is True.
 
+            error_model: str
+                The error-model affects divide-by-zero behavior.
```

Comment:
If this only effects the divide-by-zero behavior, why didn't you call it divide_by_zero_model or sth like that?